features/website/ui/dashboard/landing-settings.tsx

```
@@ -88,92 +134,157 @@ export default function LandingSettings({ hero, updateHero, businessId }: HeroSe
+
+          <div className="space-y-3">
+            <p className="text-sm font-medium text-foreground">Live preview from your site theme</p>
+            <div
+              className={cn(
+                "w-full",
+                `${resolvedTheme}-theme`
+              )}
+              style={themeColorsToCssVariables(themeColors) as CSSProperties}
+            >
+              {themeOverlayCss && (
+                <style dangerouslySetInnerHTML={{ __html: themeOverlayCss }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)